Client/Ecosia/Extensions/HomepageViewController+Ecosia.swift

```diff
@@ -7,38 +7,24 @@ import Core
 
 protocol HomepageViewControllerDelegate: AnyObject {
     func homeDidTapSearchButton(_ home: HomepageViewController)
-    func homeDidPressPersonalCounter(_ home: HomepageViewController, completion: (() -> Void)?)
+    func showSettingsWithDeeplink(to destination: AppSettingsDeeplinkOption)
```

Comment:
Sure, makes sense, will change it 👍

Reply:
Btw, this method was actually also from `BrowserViewController`, so I had to do a sort of forwarding inside `BrowserViewController+Ecosia` so that it also conforms to it and so that no renaming of Firefox code is needed. Still I don't mind and might find it even better so that it is explicit the conformance, the only downside is that `BrowserViewController` has two methods with very similar names... Let me know what you think about that

Reply:
> the only downside is that BrowserViewController has two methods with very similar names

I personally don't see that as a downside. To me will look much cleaner in terms of responsibility

Reply:
Let's leave it as it then 💪 - it's ok as perhaps FF codebase updated it